src/odemis/gui/comp/miccanvas.py

```diff
@@ -983,6 +983,9 @@ def __init__(self, *args, **kwargs):
         # This canvas can have a special overlay for tracking position history
         self.history_overlay = None
 
+        # Disable certain tools in overview. Don't list ROA, ROI, or spot mode
+        self.allowed_modes = guimodel.ALL_TOOL_MODES.difference((guimodel.TOOL_SPOT, guimodel.TOOL_ROA, guimodel.TOOL_ROI))
```

Comment:
For manipulating sets, it's nicer to use operators. You can write this line more clearly with:
`guimodel.ALL_TOOL_MODES - {guimodel.TOOL_SPOT, guimodel.TOOL_ROA, guimodel.TOOL_ROI}`